.env.sample

```diff
@@ -0,0 +1 @@
+SIGNABLE_API_KEY = api_key
```

Comment:
This is here as an example of what anyone who works on this repo locally will need to do to get the tests passing - create a `.env` file and add a valid `SIGNABLE_API_KEY` to it.
There is functionality in the gem that reads the API key from the config in our `api` repo, so tests won't pass when this repo is worked on locally unless a `.env` file is setup with a valid key in it.

Reply:
please set this in `.env.test` if that is what its for?.. better still use test vars and not the ENV

Reply:
The `Client` class has this complexity:
```
      def initialize
        self.class.base_uri "https://#{Signable.configuration.base_url}/v1"
        self.class.basic_auth Signable.configuration.api_key, ''
        self.class.default_options[:verify] = false
      end
```
We were unable to find a way of stubbing, for test runs, the API key or base URL that `Client` gets from `Signable.configuration`.

Reply:
yes this is just the sample  dotenv file. similarly how we have it on the api. but I agree it should probably be limited to test environment. so the sample should be called `.env.test.sample`

Reply:
File renamed in: [40e0ee](https://github.com/smartpension/signable/pull/12/commits/40e0ee6887f4e8ae78ca3892809ba82c331c32e0)